packages/client/lib/ProofOfPossessionBuilder.ts

```diff
@@ -8,7 +7,9 @@ import {
   PROOF_CANT_BE_CONSTRUCTED,
   ProofOfPossession,
   ProofOfPossessionCallbacks,
-} from './types';
+} from '@sphereon/openid4vci-common/lib';
```

Comment:
This is referencing a dir in the package, suggesting the exports are not in order. Should not have to include /lib

Reply:
exports were fine. Intellij auto-refactor upone move added the folder.
Removed the lib part from path.